lib/engine/game/g_18_uruguay/nationalization.rb

```diff
@@ -153,8 +123,9 @@ def start_merge(originatior)
           @fce.floatable = true
           @stock_market.set_par(@fce, fce_share_price)
           after_par(@fce)
-
-          acquire_shares_in_fce(@fce, @merge_data)
+          @merge_data[:corp_share_sum] = @merge_data[:holders].reduce(0) do |sum, holder|
+            sum + ((affected_shares(holder, @merge_data[:corps]).sum(&:percent) / 20).to_i * 10)
+          end
```

Comment:
Do you need to use `reduce` here? Would `sum` be simpler?
```suggestion
          @merge_data[:corp_share_sum] = @merge_data[:holders].sum do |holder|
            (affected_shares(holder, @merge_data[:corps]).sum(&:percent) / 20).to_i * 10
          end
```